### ISSUE TITLE
Regarding the inability to adapt to custom resolutions when launching games through GIMI

### ISSUE THREAD

**Issue by Crostia** [SpectrumQT/XXMI-Launcher#38]
The reason is that I unlocked a frame rate of 120 in the regular settings and set the resolution parameter in the command. But it cannot be transmitted to the independent launcher unlockfps_nc of the game. I hope to add plugins for automatic harp playing and free camera in the XXXMI launcher.

**Comment by SpectrumQT** [comment]
I've synced my GI FPS Unlocker fork with main and pushed the v3.3.2 update. Now its `fps_config.json` (located in `XXMI Launcher\Resources\Packages\GI-FPS-Unlocker`) supports `AdditionalCommandLine` option. For now, please configure it manually. Later I'll make launcher pass Launch Options from Settings to this json automatically.

**Comment by Crostia** [comment]
![1](https://github.com/user-attachments/assets/997b0482-20e7-425e-8756-26474439846b)
![2](https://github.com/user-attachments/assets/088356f2-6698-4247-9ee1-5d5ec4aae83b)
I have tested and it still doesn't work. The reason why the custom resolution has not been applied is not due to the parameter "fps_config. json" not being set, but rather an Inject operation before the XXMI launcher starts the game. I discovered it after multiple observations. If the previous Inject operation is not fixed, the custom resolution parameter in "fps_config. json" will not be applicable to the game. I have a simple fix: 1. If you start the game with Foce 120FPS, 2. Stop the previous injection operation of the XXMI launcher. 3. Write d3d11.dll to the parameter fps_fig.json. 4. Start the injection through unlockfps_nc.
